app/layout/statics/button-link/component.tsx

```diff
@@ -1,21 +1,45 @@
 import React from 'react';
 
 import Icon from 'components/icon';
+import Link from 'next/link';
 
 import ARROW_RIGHT_2_SVG from 'svgs/ui/arrow-right-2.svg?sprite';
 
 export interface StaticButtonLinkProps {
-
+  caption: string;
+  href: string;
+  internal?: boolean;
 }
 
-export const StaticButtonLink: React.FC<StaticButtonLinkProps> = () => {
-  return (
-    <a className="flex flex-row items-center" href="/">
-      <p className="mr-4 text-lg text-primary-500">Learn More</p>
+export const StaticButtonLink: React.FC<StaticButtonLinkProps> = ({
+  caption,
+  href,
+  internal = false,
+}: StaticButtonLinkProps) => {
+  const children = (
+    <>
+      <p className="mr-4 text-lg text-primary-500">{caption}</p>
       <div className="flex items-center justify-center bg-transparent border rounded-full h-7 w-7 border-primary-500">
         <Icon icon={ARROW_RIGHT_2_SVG} className="w-3 h-3 text-primary-500" />
       </div>
-    </a>
+    </>
+  );
+
+  return (
+    <>
+      {internal && (
+        <a className="flex flex-row items-center cursor-pointer" href={href}>
+          {children}
+        </a>
+      )}
+      {!internal && (
```

Comment:
when you say internal, you mean external right? Change the name of this prop to `external`